package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "order-form",
-  "version": "1.0.1",
+  "version": "1.1.0",
```

Comment:
I am not sure if this should be a patch or minor.

Reply:
I'd probably treat this as a patch. However, we use git versioning for the actual applications so this is less important here but it doesn't do any harm to update the `package.json` version as well.

Git versioning should automatically apply a patch version although you can also explicitly state that the patch version should be increased. There a couple of ways to do this but I tend to find the simplest is to include `+semver:patch` in your commit message body – this is another thing that I need to document.

Reply:
I have updated it to a patch, looks like a new package-lock was generated, I expect that is fine.